src/components/TradeWidget/Price.tsx

```diff
@@ -58,12 +61,11 @@ const Wrapper = styled.div`
   }
 `
 
-export const PriceInputBox = styled.div`
-  display: flex;
+export const PriceInputBox = styled.div<{ hidden?: boolean }>`
+  display: ${(props): string => (props.hidden ? 'none' : 'flex')};
   flex-flow: column nowrap;
   margin: 0;
-  width: 50%;
-  width: calc(50% - 0.8rem);
+  width: 100%;
```

Comment:
Since `PriceInputBox` is exported and used elsewhere in the app, this width change causes issues elsewhere.

Reply:
See screenshot by Pimato

Reply:
Yep, we notice yesterday. We'll fix it. Thanks for the heads up